### DISCUSSION TITLE
Command line arguments

### DISCUSSION THREAD

**Discussion by josephversace** [erigontech/erigon#2983] [Q&A] [answered]
Hi there,

I can't seem to find the command line arguments for running an archive node (with debug) for ethtx. Can anyone provide? Thanks.

**Comment by AskAlexSharov** [comment]
Probably it enabled by default?

**Comment by AskAlexSharov** [comment, marked as answer]
We also have trace_* methods from OpenEthereum

**Comment by josephversace** [comment]
Thank you
